.github/settings.yml

```diff
@@ -0,0 +1,114 @@
+###
```

Comment:
I would like to discuss moving this file to a central location, today we are managing some of these settings in zuul.